ignite/contrib/metrics/average_precision.py

```diff
@@ -33,14 +33,20 @@ class AveragePrecision(EpochMetric):
         AveragePrecision expects y to be comprised of 0's and 1's. y_pred must either be probability estimates or
         confidence values. To apply an activation to y_pred, use output_transform as shown below:
 
-        .. code-block:: python
+        .. testcode::
 
-            def activated_output_transform(output):
-                y_pred, y = output
-                y_pred = torch.softmax(y_pred, dim=1)
-                return y_pred, y
+            y_pred = torch.Tensor([[0.79, 0.21], [0.30, 0.70], [0.46, 0.54], [0.16, 0.84]])
+            y_true = torch.tensor([[1, 1], [1, 1], [0, 1], [0, 1]])
 
-            avg_precision = AveragePrecision(activated_output_transform)
+            avg_precision = AveragePrecision()
+            #The ``output_transform`` arg of the metric can be used to perform a softmax on the ``y_pred``.
```

Comment:
Thinking about it, it is worth to keep the current example to show the way to respect the data format 

```python 
AveragePrecision expects y to be comprised of 0's and 1's. y_pred must either be probability estimates or
        confidence values. To apply an activation to y_pred, use output_transform as shown below:

        .. code-block:: python

            def activated_output_transform(output):
                y_pred, y = output
                y_pred = torch.softmax(y_pred, dim=1)
                return y_pred, y

            avg_precision = AveragePrecision(activated_output_transform)
```